api-spec/examples.md

```diff
@@ -3,53 +3,84 @@
 A STAC API supports both GET and POST requests. It is best to use POST when using the intersects query for two reasons:
 
 1. the size limit for a GET request is less than that of POST, so if a complex geometry is used GET may fail.
-2. The parameters for a GET request must be escaped properly, making it more difficult to construct when using JSON parameters (such as intersect)
+2. The parameters for a GET request must be escaped properly, making it more difficult to construct when using JSON 
+parameters (such as intersect)
 
 ## Core examples
 
+Requests all the data in the collection that is in New Zealand:
```

Comment:
maybe call this explicitly `Items` rather than data -- since technically all STAC entities are metadata

Reply:
Sounds good